python/ray/serve/_private/autoscaling_state.py

```diff
@@ -367,6 +371,76 @@ def get_total_num_requests(self) -> float:
         return total_requests
 
 
+class ApplicationAutoscalingState:
+    """Manages autoscaling for a single application."""
+
+    def __init__(
+        self,
+        app_name: ApplicationName,
+        deployment_autoscaling_states: Dict[DeploymentID, DeploymentAutoscalingState],
+    ):
+        self._app_name = app_name
+        self._deployment_autoscaling_states = deployment_autoscaling_states
+        self._config = None
+        self._policy = None
+
+    def register(self, config: ServeApplicationSchema) -> int:
+        self._config = config
+        self._policy = self._config.autoscaling_policy
+        self._policy_state = {}
```

Comment:
what is the type of `self._policy_state`?

Reply:
should be Dict[str, Dict]